pallets/asset/src/lib.rs

```diff
@@ -212,6 +212,12 @@ impl<T: AsRef<[u8]>> From<T> for FundingRoundName {
     }
 }
 
+impl Default for FundingRoundName {
+    fn default() -> Self {
+        FundingRoundName("GenesisRound".as_bytes().to_vec())
```

Comment:
I think the default should be blank not GenesisRound.

Reply:
Fixed

Reply:
I make this because I need to emit the funding round name and the amount raised in the funding round in `Issued` event from `create_asset()` function. Or I need to make round optional parameter in the event and passed 0 as the issued_amount_in_current_round.